humble/distribution.yaml

```diff
@@ -6442,6 +6442,21 @@ repositories:
       url: https://github.com/ros2/sros2.git
       version: humble
     status: developed
+  stcamera_components:
+    doc:
+      type: git
+      url: https://github.com/ose-support-ros/stcamera_ros2.git
+      version: humble
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ose-support-ros/stcamera_ros2.git
+      version: 1.0.1
```

Comment:
This whole section is not correct for a release.  In order to do a release, you'll need to create a `ros2-gbp` repository and use `bloom-release` to generate the release metadata.  See http://docs.ros.org/en/humble/How-To-Guides/Releasing/Releasing-a-Package.html for detailed information.

If you'd prefer not to do all of that now, you can just add in the `doc` and `source` entries for now, meaning that the package will be available for indexing and document generation.  A binary package won't be available until you follow through with the release instructions, though.

Reply:
Thank you for your explanation. Since building this package requires a separate manual library installation, we remove the release entry.